libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -236,9 +236,7 @@ internal class RouteRefreshController(
             routeProgressData.legIndex,
             routeProgressData.routeGeometryIndex,
             routeProgressData.legGeometryIndex,
-            hashMapOf<String, String>().apply {
-                putAll(evRefreshDataProvider.get(route.routeOptions))
```

Comment:
I did it in order to show that technically it can be not only EV data. But OK, no big deal.